declarative-examples/jenkinsfile-examples/mavenDocker.groovy

```diff
@@ -0,0 +1,102 @@
+pipeline {
+
+  //Run everything on an agent that has docker daemon and git installed
```

Comment:
Perhaps should clarify that this needs to be set up in Jenkins and doesn't just magically exist?